roles/vpc/tasks/router.yaml

```diff
@@ -0,0 +1,28 @@
+---
+# upstream expects enable_snat to be defaulted to true, but in OTC it is not that
+# therefore it is required to enable snat manually later or use heat
+- name: Create Router
+  openstack.cloud.router:
+    name: "{{ router_name }}"
+    state: "{{ state }}"
+    network: "{{ external_network_name }}"
+    enable_snat: "{{ enable_snat | default(omit) }}"
```

Comment:
but in EU-NL we won't have shared snat at all

Reply:
yupp, and this will not set it at all unless really variable set